client/components/dialog/README.md

```diff
@@ -66,22 +66,24 @@ You can attach `onClick` handlers for dialog buttons. The `onClick` handler will
 called will close the dialog the dialog button is a member of.
 
 ```js
```

Comment:
We could do with using JSX here - outside the aims of this PR though so FFTI

Reply:
That actually causes syntax highlighting to go away in my editor (Atom) :slightly_frowning_face:

Reply:
Ohh that's interesting - I always push for ` ```jsx` in code reviews but perhaps `js` should be generally preferred since it's much more widely supported